src/payment/payment-methods/apple-pay/service.js

```diff
@@ -107,7 +119,7 @@ export function checkout(basket) {
         .catch((error) => {
           // eslint-disable-next-line no-param-reassign
           error.code = 'apple-pay-authorization-failure';
-          logApiClientError(error, {
+          logError(error, {
```

Comment:
Does logError accept a messagePrefix still?  Looks like it doesn't do anything special with it, it'll just pass it through like any other `customAttributes` key.  Is that okay?

Reply:
It doesn't do anything special with it. We can make a subsequent update to frontend-platform to prepend messagePrefixes if we want. It's one of the improvements Robert had suggested for logging I think